salt/modules/yumpkg.py

```diff
@@ -258,6 +263,9 @@ def _get_options(**kwargs):
         log.info('Adding branch \'%s\'', branch)
         ret.append('--branch={0}'.format(branch))
 
+    for item in setopt:
+        ret.extend(['--setopt', six.text_type(item)])
```

Comment:
Yes, I tested using multiple `--setopt` on the CLI and it worked like that. I just preferred to do it that way to avoid any confusion with more than one `=` sign, though it does work both ways. Really just a preference thing, it would have worked the other way too.

Reply:
thanks for confirming :)